src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -257,6 +257,7 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 			this._lastUserDataTimestamp = Date.now();
 		}));
 		this._register(xterm.onScroll(() => this.hideSuggestWidget(true)));
+		this._register(xterm.onResize(() => this.hideSuggestWidget(true)));
```

Comment:
This fix seems risky, we want the suggest widget to stick around when resizing, just like it does in the editor. There can also be some unexpected resize events that happen in the terminal which would unexpectedly hide it. The proper fix for this imo is if resizing, make sure it's not shown if it was not visible before the resize.